### PR TITLE
lezzet.com.tr [leftovers, rule added]

### PR DESCRIPTION
**Pull request by stanislav-atr**
#62072

### CODE REVIEW

File: TurkishFilter/sections/specific.txt [hunk 1]
```diff
@@ -82,6 +82,7 @@ oyungezer.com.tr##.leftLink
 oyungezer.com.tr##.rightLink
 lezzet.com.tr##.row > div[style="margin-top:-10px"]
 lezzet.com.tr##.banner-box
+lezzet.com##div[id^="medyanet/"]
```
Comment (zloyden) [inline TurkishFilter/sections/specific.txt:85]: `lezzet.com.tr`

**Review by zloyden — CHANGES_REQUESTED** [review]

File: TurkishFilter/sections/specific.txt (hunk 1 above)
Comment (zloyden) [reply TurkishFilter/sections/specific.txt:85]: Also remove these leftovers
<details><summary>Screenshot:</summary>

image

</details><br/>

Comment (stanislav-atr) [reply TurkishFilter/sections/specific.txt:85]: Don't see these.
image

Comment (zloyden) [reply TurkishFilter/sections/specific.txt]: <details><summary>Screenshots:</summary>

image
image
image

</details><br/>

Comment (stanislav-atr) [reply TurkishFilter/sections/specific.txt]: It seems these are blocked with
image
on my side. Should i add these rules to Turkish filters too?

Comment (zloyden) [reply TurkishFilter/sections/specific.txt]: Add a specific rule `lezzet.com.tr##ins[data-revive-zoneid]` to the Turkish filter

**Review by zloyden — APPROVED** [review]

**Pull request merged**